peppy/simple_attr_map.py

```diff
@@ -46,6 +46,9 @@ def __getattr__(self, item):
         except KeyError:
             raise AttributeError(f"Attribute not found: {item}")
 
+    def __eq__(self, other: "SimpleAttMap"):
+        return self._mapped_attr == other._mapped_attr
```

Comment:
Tests for these? Is the `mapped_attr` just a dictionary?

Reply:
Yeah, I added tests. it is just dict:)